.github/workflows/ci.yml

```diff
@@ -0,0 +1,44 @@
+name: CI
+
+on:
+  release:
+    types: [created]
+  push:
+  schedule:
+    # Runs every Thursday at 20:23 GMT to avoid bit rot
+    - cron: "20 23 * * 4"
```

Comment:
Does this mean a new release will be created automatically every Thursday? And if so, what will be the release number?

Reply:
We will build the app every Thursday for a test, but the release is guarded by this condition: ```      if: startsWith(github.ref, 'refs/tags/')
```
That means, only if we're on a tag it will try to publish the release. 
In an automated build, this condition is `false` and we should be good. Thanks for raising that concern, though. 😉

Reply:
Sounds good!